### DISCUSSION TITLE
export json file

### DISCUSSION THREAD

**Discussion by ljs2905** [Authenticator-Extension/Authenticator#1037] [Ideas]
Please, add the option to export as a json file and enable making popup default mode.

**Comment by Sneezry** [comment]
Could you share more context why you prefer the JSON backup file? The standard format lets you quickly transfer your data between different 2FA clients.

**Comment by ljs2905** [comment]
Dear  Zhe Li,  I find your  AUthenticator extension really handy and useful with the Edge browser, but it means I do need a browser and it is not compatible with android applications (to the best of my knowledge). For that I use the free otp application FreeOTP+ versao 3.0 (21), which I’d rateher use than The Google authenticaticator,  as yours works far better, and  which can import both samll txt files and longer json files. If I import txt files, I’ll need to split your backup file into smaller ones, or convert the full txt file into a json file by hand, as I haven’t been able to find and aplication that does it for me. Naturally, I’m unaware of an  application of yours for android. Is there one?   https://f-droid.org/packages/org.liberty.android.freeotpplus/   Kind regards,  Laerte J SIlva    Enviado do Email para Windows De: Zhe LiEnviado:31 March 2023 11:15Para: Authenticator-Extension/AuthenticatorCc:ljs2905; AuthorAssunto: Re: [Authenticator-Extension/Authenticator] export json file (Discussion #1037) Could you share more context why you prefer the JSON backup file? The standard format lets you quickly transfer your data between different 2FA clients.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***> Não contém vírus.www.avast.com

**Comment by Sneezry** [comment]
Hi @ljs2905 sorry for the late reply. I missed your message. FreeOTP+ should support the standard backup format (https://authenticator.cc/docs/en/otp-backup). Could you explain more about why you need to split the backup file into multiple smaller ones? Is there any content size limit of FreeOTP+?
